parsers/ocaml.c

```diff
@@ -1809,9 +2024,20 @@ static void localScope (vString * const ident, ocaToken what)
 static void computeModuleName ( void )
 {
 	/* in Ocaml the file name define a module.
-	 * so we define a module =)
+	 * so we define a module if the file has
+	 * things in it. =)
 	 */
 	const char *filename = getSourceFileName ();
+
+	FILE *file = fopen (filename, "r");
+	fseek (file, 0, SEEK_END);
+	long fileSize = ftell (file);
+	fclose (file);	// CLOSE!
+	if (fileSize == 0)
+	{
+		return;
+	}
```

Comment:
please, do not play with IO like this.  Not only it adds some (probably unnecessary) overhead, but it also complicates possible future move to a non-file-backed IO layer.  So please use another way of determining if the file is empty.

BTW, it can even be better, as this only finds empty files if they are truly empty, but not if they contain e.g. a newline, or some whitespace, or a comment, etc.
What about rather checking if some tags were emitted, or if it's not good enough, if a token was generated?  And emit the module tag at the end of the processing (which should probably be alright, as I don't think CTags has any promise on the tag ordering -- does it?